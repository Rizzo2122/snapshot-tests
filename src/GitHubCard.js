import React from "react"
import Card from "react-bootstrap/Card"

function GitHubCard() {
    return (
        <div className="card">
            <Card style={{ width: "18rem" }}>
                <Card.Img variant="top" src="https://avatars.githubusercontent.com/u/112102445?v=4" />
                <Card.Body>
                    <Card.Title>Rizzo2122</Card.Title>
                    <Card.Text>
                        I'm a 19 year old coding student
                    </Card.Text>
                </Card.Body>
            </Card>
        </div>
    );
}

export default GitHubCard