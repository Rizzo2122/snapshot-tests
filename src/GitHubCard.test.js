import GitHubCard from "./GitHubCard";
import Renderer from "react-test-renderer";

test('renders a snapshot', () => {
    const tree = Renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot()
})