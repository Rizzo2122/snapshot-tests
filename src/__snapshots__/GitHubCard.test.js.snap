// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "18rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="https://avatars.githubusercontent.com/u/112102445?v=4"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Rizzo2122
      </div>
      <p
        className="card-text"
      >
        I'm a 19 year old coding student
      </p>
    </div>
  </div>
</div>
`;
